src/submit_jobs.py

```diff
@@ -20,7 +20,7 @@
 # Default values for jobs, used per job if not explicitly set in the job's
 # input JSON. CUR and YEAR values are used for partitioning and filtering
 # if they are set
-DEFAULT_VAR_MIN_YEAR = 1999
+DEFAULT_VAR_MIN_YEAR = 1800
```

Comment:
There are tables with `taxyr` as early as 1900, so we want the unbounded case (where `min_year` is NULL) to capture those earliest records.